RecoTauTag/RecoTau/test/rerunMVAIsolationOnMiniAOD_Phase2.py

```diff
@@ -2,17 +2,17 @@
 
 process = cms.Process("rerunMVAIsolationOnMiniAODPhase2")
 
-process.load('Configuration.Geometry.GeometryExtended2026D41Reco_cff')
+process.load('Configuration.Geometry.GeometryExtended2026D49Reco_cff')
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
 from Configuration.AlCa.GlobalTag import GlobalTag
-process.GlobalTag = GlobalTag(process.GlobalTag, '106X_upgrade2023_realistic_v2')
+process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase2_realistic_T15')
```

Comment:
shouldn't the input file be updated as well?
it seems inconsistent to read D35 data with D49 settings

```
	D35 = T6+C4+M2+I5+O2+F2
	D41 = T14+C8+M3+I9+O3+F2
	D49 = T15+C9+M4+I10+O4+F2
```

Reply:
I suspect this test makes **no use of GT or geometry whatsoever**, otherwise would have crashed.
Indeed if I comment these lines, 

```diff
iff --git a/RecoTauTag/RecoTau/test/rerunMVAIsolationOnMiniAOD_Phase2.py b/RecoTauTag/RecoTau/test/rerunMVAIsolationOnMiniAOD_Phase2.py
index 223a579c8e1..caaa94a0ca3 100644
--- a/RecoTauTag/RecoTau/test/rerunMVAIsolationOnMiniAOD_Phase2.py
+++ b/RecoTauTag/RecoTau/test/rerunMVAIsolationOnMiniAOD_Phase2.py
@@ -2,17 +2,17 @@ import FWCore.ParameterSet.Config as cms
 
 process = cms.Process("rerunMVAIsolationOnMiniAODPhase2")
 
-process.load('Configuration.Geometry.GeometryExtended2026D41Reco_cff')
-process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
-from Configuration.AlCa.GlobalTag import GlobalTag
-process.GlobalTag = GlobalTag(process.GlobalTag, '106X_upgrade2023_realistic_v2')
+#process.load('Configuration.Geometry.GeometryExtended2026D41Reco_cff')
+#process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+#from Configuration.AlCa.GlobalTag import GlobalTag
+#process.GlobalTag = GlobalTag(process.GlobalTag, '106X_upgrade2023_realistic_v2')
 process.load("Configuration.StandardSequences.MagneticField_cff")
 
 #process.Tracer = cms.Service("Tracer")
 process.load("FWCore.MessageService.MessageLogger_cfi")
-process.MessageLogger.cerr.FwkReport.reportEvery = 1000
+process.MessageLogger.cerr.FwkReport.reportEvery = 1
 process.maxEvents = cms.untracked.PSet(
-    input = cms.untracked.int32(10000)
+    input = cms.untracked.int32(10)
 )

```

this piece of configuration still runs.
By the way I see that below it is loading:
```python
process.load('RecoTauTag.Configuration.loadRecoTauTagMVAsFromPrepDB_cfi')
```

isn't this in direct violation of issue https://github.com/cms-sw/cmssw/issues/27393 ?

Reply:
Yes, indeed payloads of this tauID are still not mapped via a global tag. It is due to some misunderstanding between tau developers. We are going to add the payloads to global tag queues ASAP. But, until it is done we should deal with current solution.

Reply:
BTW, does the failure indeed related with this small update or it was just incidental? Or in other words - do we need update this PR in some way, e.g. use different GT / geometry combination (is so what you propose) or it is fine as it is?

Reply:
The addOnTest failure seen before is unrelated and incidental. As for the geometry and GT, my suggestion is just to delete the lines as this are not used for the moment so that you can forget about constantly updating it as the phase-2 geometry evolves.

Reply:
OK, thanks for suggestions. I will add a comment that GT should be added back when related payloads are governed by it.